### PR TITLE
Avoid setting "clientX" and "clientY" to "undefined" in drag() call. Fix swapped x/y coordings in move()

### PR DESCRIPTION
**Pull request by nils4cosee**
This PR actually addresses two issues:

1) Issue #80 : "top" and "left" are swapped in the call to "dragstart" within "move()"
2) Issue #60 : When we add eventlisteners for that evuluate the "clientX" and "clientY" of the move-events,
    we notice that those values are null. This is probably because they are set to "undefined" in the trigger() call.
    This change makes sure that the properties are not there (rather than being undefined). This means that cypress
    fills in the correct values depending on the elements location.

Please tell me, if you need any changes or additional things for the PR.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -79,8 +76,7 @@ const DragSimulator = {
                 this.target.trigger('pointerup', {
                   which: 1,
                   button: 0,
-                  clientX,
-                  clientY,
+                  clientPosition,
```
Comment (lvonlanthen) [inline index.js:79]: I think the `clientPosition` object should also be spreaded here:
```suggestion
                  ...clientPosition,
```

**Review by lvonlanthen — CHANGES_REQUESTED** [review]
🎉 Thank you very much for your contribution, it is much appreciated! The changes look good, I just added a small comment. Could you please fix this and then I think this PR is ready to be merged.

File: index.js (hunk 1 above)
Comment (nils4cosee) [reply index.js]: You are right. I updated the PR. Thanks for taking care of this project.

**Review by lvonlanthen — APPROVED** [review]
👍🏼 Thank you very much!

**Pull request merged**